documentation/TechnicalDesignDocument.tex

```diff
@@ -184,6 +184,24 @@
     When deleting an object, the user selects an object using a boundingbox and a ray on the press of a button. This process is explained above: \href{\clickup{2e29e}}{\#2e29e}.
     When this occurs the object is turned slightly transparent to function as a preview.
     While an object is selected for deletion, the camera is locked.
+    A second click will delete the object by removing it from the list of instances in the world.
+    If the click hits another object, that object becomes selected instead.
+    If it clicks elsewhere it deselects entirely.
+
+    \myparagraph{\href{\clickup{2e294}}}{\#2e294} As a MGD or MG I want the blocks to connect while I am dragging it within a certain range
+    Using the bounding box you can find which object you are close to fairly easily.
+    However, there is no functionality in libGDX for finding the face of an object.
+    To remedy this a new addition was made to the SelectableInstance: Faces.
+    Every SelectableInstance has a list of Faces/Joints which contain a vector3 which states the location of the joints.
```

Comment:
In my opinion you should not write Joints here. Faces are the graphical / math term for '2D' shapes that make up a 3D shape. Joints are a marama term, and only certain faces are joint, not all.